src/components/dropdown/dropdown.stories.mdx

```diff
@@ -0,0 +1,27 @@
+import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs';
+import { Dropdown, DropdownOption } from './dropdown';
+import { Icon } from '../../new-components/icon/icon';
+
+<Meta
+  title="Components/Dropdown"
+  component={{ Dropdown }}/>
```

Comment:
storybook has the concept of `subcomponents`. Is this relevant here? I don't know, because actually you can just pass  `DropdownOption` to the args table anyway?

https://storybook.js.org/docs/react/workflows/stories-for-multiple-components

Reply:
Since I'm now using the interfaces from ReachUI, the args table isn't working anymore. I spent some time trying to fix it but it didn't work. Do you have a good suggestion on how to best display the args from an external component?